### DISCUSSION TITLE
How to move to sidebar to the right?

### DISCUSSION THREAD

**Discussion by listvin** [mbnuqw/sidebery#1693] [Q&A] [answered]
Sidebar is currently on the left, is it possible to move it to the right?
I use ff primarily on left monitor, so neck kinda hurts (:

I searched through settings, checked discussions and issues, seems nothing on topic.

Extension is extremely cool as it is, THANK YOU for great work. I feel kinda shame for asking more (:

**Comment by albino1** [comment, marked as answer]
This is already built into Firefox:

![sb1](https://github.com/mbnuqw/sidebery/assets/12164957/125f1544-9c7e-490f-94c7-887a7d2e7b19)
